components/AddressBook.php

```diff
@@ -32,6 +33,27 @@ public function onLoadAddForm()
         return ['#address-book' => $this->renderPartial('@form')];
     }
 
+    public function setDefaultAddress()
```

Comment:
call it onSetDefaultAddress() to follow the conventions

Reply:
Ignore this.

Reply:
I didn't change it - left as per comment below. Let me know if you want it changed.